### PR TITLE
MAX_HEAP config was not good && update doc

### PR DESCRIPTION
**Pull request by guizmaii**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -65,7 +65,7 @@ process, which runs as UID 200.
 
   * `CONTEXT_PATH`, passed as -Dnexus-webapp-context-path.  This is used to define the
   URL which Nexus is accessed.
-  * `MAX_HEAP`, passed as -Xmx.  Defaults to `1g`.
+  * `MAX_HEAP`, passed as -Xmx.  Defaults to `768m`.
```
Comment (brianwawok) [inline README.md]: why not keep it at 1024m ?

**Comment by guizmaii** [comment]
Because it's not the default value you put in your configuration

File: README.md (hunk 1 above)
Comment (guizmaii) [inline README.md]: Because it's not the default value you put in your configuration

**Comment by bhamail** [comment]
+1, these changes make docs match the actual config.

**Pull request merged**